editor/src/clj/editor/editor_extensions.clj

```diff
@@ -563,9 +563,9 @@
 (defn- re-create-ext-state [initial-state evaluation-context]
   (let [{:keys [rt display-output!]} initial-state]
     (->> (e/concat
+           [bundle-editor-script-prototype]
            (:library-prototypes initial-state)
-           (:project-prototypes initial-state)
-           [bundle-editor-script-prototype])
+           (:project-prototypes initial-state))
```

Comment:
Not related to the issue: loading the built-in editor script first displays the usual bundle menu items before the ones defined by extensions.